plugins/scaffolder-backend-module-gitlab/src/commonGitlabConfig.ts

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* We want to maintain the same information as an enum, so we disable the redeclaration warning */
+/* eslint-disable @typescript-eslint/no-redeclare */
```

Comment:
Could you put these only over the lines that need them, rather than the entire file?

Reply:
Unfortunately I don't think that's possible because `@public` and the eslint line both need to be the line above the declaration otherwise I get an error.